android/fastlane/Fastfile

```diff
@@ -126,9 +126,12 @@ platform :android do
             version_code: version_code,
             version_name: version_name,
             package_name: "app.lunes",
-            metadata_path: "./android/fastlane/lunes/metadata",
+            metadata_path: "./fastlane/metadata",
```

Comment:
Shouldn't it be `./android/fastlane/metadata`?

Reply:
no, because the working directory is /android/
Tested it and is working :)

Reply:
Ah okay. Any specific reason why you choose android as working directory but ios not?

Reply:
For clarification: The release-notes for android are working, the prod_delivery is not tested as not possible right now, therefore the comment to do this in mid of september.

Reply:
yeah, because this is an android job? For ios the working_directory is `ios`

Reply:
But the metadata path for ios includes `ios` while the metadata path for android does not include `android`, that's all I wanted to say :D

Reply:
oke, will fix this.